### PR TITLE
[MIG] sale_timesheet_ux: Migration to 16.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/sale/pull/599).

File: sale_timesheet_ux/models/project_task.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-from odoo import models
+from odoo import models, fields
 
 
 class ProjectTask(models.Model):
 
     _inherit = 'project.task'
 
+    order_id = fields.Many2one('sale.order', 'Sales Order', store=True, help="Sales order to which the task is linked.")
```
Comment (JrAdhoc) [inline sale_timesheet_ux/models/project_task.py:8]: @matiasperalta1 Mati, el build de runbot da un error que te dejo acá con este campo:
Two fields (order_id, sale_order_id) of project.task() have the same label: Sales Order. [Modules: sale_timesheet_ux and sale_timesheet]

**Review by JrAdhoc — CHANGES_REQUESTED** [review]

**Comment by JrAdhoc** [comment]
@matiasperalta1 
Tmb dice que hay que hacer un rebase a origin 16.0

**Comment by JrAdhoc** [comment]
@roboadhoc r+

**Comment by JrAdhoc** [comment]
@roboadhoc r+ override=ci/oba_without_tests p=0

**Comment by roboadhoc** [comment]
I'm sorry, @JrAdhoc:
- this PR is already reviewed, reviewing it again is useless.
- you are not allowed to override this status.

**Pull request closed without merging**